vendor/github.com/google/go-github/v53/github/github-accessors.go

```diff
@@ -166,6 +166,22 @@ func (a *ActionsPermissionsRepository) GetEnabled() bool {
 	return *a.Enabled
 }
 
+// GetDefaultWorkflowPermissions returns the DefaultWorkflowPermissions field if it's non-nil, zero value otherwise.
```

Comment:
Did you make these updates to vendored files manually? How did this come about without associated go.mod and go.sum files?